script2.js

```diff
@@ -0,0 +1,94 @@
+const API = 'https://pokeapi.co/api/v2/pokemon/'
+const wrapper = document.querySelector('wrapper')
+const main = document.querySelector('main')
+const pokemonOne = document.querySelector('.pokemonOne')
+const backCards = Array.from(document.querySelectorAll('.backCard'))
+const next = document.querySelector('.next')
+const previous = document.querySelector('.previous')
+const page = document.querySelector('.currentPage');
+let localDataBase = ['empty'];
+let pokemonsFetchesSoFar = 0;
+let currentPage = 0;
+let c = pokemonsFetchesSoFar; // counter
+let allPages = [];
+
+async function fetchPokemons() {
+    let pokemonsFetched = [];
+
+    for (let i = 0; i < 3; i++) {
+        c++
+        pokemonsFetched.push(await (await fetch(API + c)).json());
+    }
+    localDataBase.push(pokemonsFetched);
+    return pokemonsFetched
+}
+
+async function domUpdate(fn, direction) {
+
+    if (direction === 'next') {
+        currentPage++;
+    } else if (direction === 'previous' && currentPage > 1) {
+        currentPage--;
+    } else {
+        return;
+    }
+
+    if (currentPage >= localDataBase.length) {
+        await fn();
+    }
+
+    pokemonOne.innerHTML = ` <img src="${localDataBase[currentPage][1].sprites.other.dream_world.front_default}"
+    alt="${localDataBase[currentPage][1].name}">
```

Comment:
Wykorzystać samozamykający tag kiedy nie ma zawartości
```suggestion
    <div class="imgOverlay"/>
    <div class="shineEffect"/>
```